tests/test_client.py

```diff
@@ -232,7 +232,13 @@ def create(self, stream, start_date, end_date=None, num_records=1):
         elif stream == 'shifts':
             employee_id = [employee['id'] for employee in self.get_all('employees', start_date)][0]
             location_id = [location['id'] for location in self.get_all('locations', start_date)][0]
-            max_end_at = max([obj['end_at'] for obj in self.get_all('shifts', start_date)])
+            all_shifts = self.get_all('shifts', start_date)
+
+            if all_shifts:
+                max_end_at = max([obj['end_at'] for obj in all_shifts])
+            else:
+                max_end_at = start_date
```

Comment:
I assume this means that shifts does not obey the start_date? Otherwise we won't be getting these reccords.

Reply:
I don't think that's true. The shifts do obey the start date. But if we are out of records the max method will fail for an empty list.

Reply:
But if we default to a max_end_at equal to the start_date, won't all records exist prior to the start_date?